proj/funcDictionary.py

```diff
@@ -0,0 +1,335 @@
+# info: some general functions:
+
+import numpy as np
+import networkx as nx
+
+#import pyspark
+from imnet import process_strings as p  # used?
+
+#import subprocess
+#import re
+#from IPython.display import HTML
+
+#import sparkhpc
+#from sparkhpc import sparkjob
+#from sparkhpc.sparkjob import LSFSparkJob
+#import findspark
+#from scipy.sparse import csr_matrix
+
+#from plotData import PlotData
+#from plotData import InnerDataLevenshtein
+#from plotData import PlotDataLevenshtein
+
+def make_nets(ls, seq): # We still need seq? What is that?
+    """
+    info: make a list "nets", where each element "net" is 
+        a list again, containing all sequences with a certain 
+        length
+    input:
+        ls:  list of length values for each sequence in seq
+        seq: array of nucleic acid sequences 
+    output: 
+        nets: list of nets, where each element "net" is 
+        a list again, containing all sequences with a certain 
+        length  
+    """
+
+    lsHist = np.histogram(ls, bins = np.arange(0, 120, 3))
+    xArgs = lsHist[1][:]
+
+    # keep in mind, that this is the average:
+
+    # info: make empty list of nets:
+    nets = list()
+    for i in range(len(lsHist[0])+1):
+        nets.append(list())
+
+    # info: insert every sequence in the net for its size.
+    #     Thus all sequences with one size are in one net
+    for seqEl in seq:
+        seqElIndex = int((len(seqEl) - lsHist[1][0])/3)
```

Comment:
Again a "harcoded" 3 that is confusing. Shouldn't that be a variable instead of a harcoded 3?

Reply:
As said, it's a biological constant. We can make it a variable, but it is as constant as pi for us.

Reply:
Ok, thanks for clarifying. Can we add "Note that dividing by 3 is used to convert nucleic acids to amino acids." somewhere in the documentation? This would be helpful for someone stumbling over this number.